week1/homework/src/fileContent.js

```diff
@@ -0,0 +1,13 @@
+function State(num) {
```

Comment:
Hmm, this is the first home work I guess you accidentally made your new branch out of the week1 branch making them follow each other history.

for avoiding that in the future, you should make checkout to master first, make a new branch, then you ca check out to it.
```bash
git checkout master
git branch <new-branch> #name it as you wish
git checkout <new-branch>
```
Or you could possibly as a shortcut do this by one command
```bash
git checkout master
# `-b` option will make a new branch and it will switch to it.
git checkout -b <new-branch> #name it as you wish
```
Or even shorter, with online you could do it as:
```bash
# `-b` option will make a new branch and it will switch to it.
# new-branch it could be `week2` and source-branch could be `master` in your case because we need to checkout from the master branch.
git checkout -b <new-branch> <source-branch>
```

Reply:
I think I'll go to do it again with a new pull request!

Reply:
No need just be aware of it the next one:)